docs/product/issues/issue-details/replay-issues/hydration-error.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Hydration Error
+sidebar_order: 50
+description: "Learn about hydration errors."
+---
+
+Hydration errors are a React-specific problem that happen when the initial client UI does not match what was rendered on the server. They can result in extra work for the browser, and a slower pageload experience for users.
+
+## Prerequisites for Seeing Hydration Error Diff Tools
+
+Hydration Errors can come into your issue stream either as regular error events, or they could be detected inside a replay session and converted into an issue by the server. Read more about
```

Comment:
```suggestion
Hydration Errors can come into your issue stream either as regular error events, or they could be detected inside a replay and converted into an issue by the server. Read more about
```

Reply:
This is really confusing :/ Can we clarify that hydration errors from 'regular error events' are filtered out via inbound filters

Reply:
The linked page `[configuring hydration errors](/platforms/javascript/session-replay/issue-types/)` does talk about the inbound filters and what the different preference options are, i think it's ok to only have that in one spot. Also, it is confusing!